features/steps/command/shell.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Running shell commands
+
+  You can run an *ad hoc* script with the following step
+  - `When I run the following script:`
+
+  Or you can run shell commands with:
+  - `I run the following (bash|zsh|fish|dash)? commands
```

Comment:
(tell me if it's the only thing to be fixed, I'll commit and push right away :))